api/services/account_service.py

```diff
@@ -793,6 +793,7 @@ def register(
         language: Optional[str] = None,
         status: Optional[AccountStatus] = None,
         is_setup: Optional[bool] = False,
+        create_workspace_required: Optional[bool] = True,
```

Comment:
bool is OK?

Reply:
Hi, thanks for the review. But sorry, I didn't quite get it what you mean.

Is any issue about bool? Could you please be more specific?

Reply:
create_workspace_required: bool = True,
seems there's no chance  None here?

Reply:
> create_workspace_required: bool = True, seems there's no chance None here?

Yeah, setting the default value of create_workspace_required to True, which ensures the existing code logic remains unchanged.

Only when the create-tenant command is run, and setting create_workspace_required to False to skip create tenant, which allowing create a tenant with custom name by -- option.